extensions/Worker.Extensions.Http.AspNetCore/src/HttpContextConverter.cs

```diff
@@ -18,7 +18,7 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
 
             if (context.TargetType == typeof(HttpRequest))
             {
-                if (context.FunctionContext.Items.TryGetValue("HttpRequestContext", out var requestContext)
+                if (context.FunctionContext.Items.TryGetValue(Constants.HttpContextKey, out var requestContext)
```

Comment:
The converter name made me think that this will populate a parameter of type `HttpContext`. Should we rename it to HttpRequestConverter ?

Reply:
Makes sense to me - will change if there are no objections.

Reply:
I think the change makes sense, but let's do it in a separate PR.